content/docs/guides/multi-target-exporter.md

```diff
@@ -399,17 +401,15 @@ The details are complicated and out of scope for this guide. Hence we will limit
 Here is the config you will use to do that. Don’t worry if this is a bit much at once, we will go through it step by step:
 
 ```yaml
+global:
+  scrape_interval:  
 scrape_configs:
-
 - job_name: blackbox-exporter-metrics
-  scrape_interval: 5s
   metrics_path: /metrics
-  static_configs:
-    - targets:
+  static_configs: # to get metrics about the exporter itself - targets:
       - localhost:9115   # for Windows and macOS replace with - host.docker.internal:9115
 
-- job_name: 'blackbox'
-  scrape_interval: 5s
+- job_name: blackbox # to get metrics about targets we want the exporter to query for us
```

Comment:
You still need to distinguish the types of probe jobs

Reply:
I know why you want that and I think this is a terrible idea for teaching.

Reply:
You want that because it fits into a bigger scheme.
But the bombardment with bigger scheme everywhere is the reason why people don’t understand what is going on and need guidance in the first place.

Reply:
I'm not saying you have to go into great depth, but the example should at least hint at how to handle this. blackbox on its own is not a good job name, unless it refers to the exporter itself.

Reply:
One option that just occurred to me is blackbox_e2e, which has the distinction without mentioning the module name.

Reply:
I understand your concern and I have changed both job names the way you proposed in the first place. `blackbox_e2e` would in my opinion be even more confusing and I want to keep the guide as clear as possible. I just want you to know and understand that while your naming scheme is great for the daily praxis, it is not necessarily ideal for learning. Making things unnecessarily hard to understand in guides for beginners is a form of gate-keeping and we have more than enough of that already. For successfully teaching physics it is better to start with classical mechanics. Quantum physics and relativity theory have to come later.